app/uk/gov/hmrc/excisemovementcontrolsystemapi/services/CorrelationIdService.scala

```diff
@@ -16,11 +16,30 @@
 
 package uk.gov.hmrc.excisemovementcontrolsystemapi.services
 
+import play.api.mvc.Request
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.http.HeaderCarrierConverter
+
 import java.util.UUID
 import javax.inject.{Inject, Singleton}
 
 @Singleton
 class CorrelationIdService @Inject() {
 
   def generateCorrelationId(): String = UUID.randomUUID().toString
```

Comment:
this either doesnt need brackets on it or the call on line 36 needs brackets adding